.tekton/operator-bundle-build.yaml

```diff
@@ -57,10 +57,22 @@ spec:
   pipelineRef:
     name: operator-bundle-pipeline
 
+  # The successful completion of the operator-bundle pipeline depends on all other ACS images
+  # having been built, since their image digests need to be injected in the bundle's
+  # ClusterServiceVersion file.
+  #
+  # To account for this inter-pipeline dependency, the cumulative tasks timeout set here
+  # needs to be kept in sync with
+  #
+  # 1. the timeouts for waiting for ACS images;
+  #    see  spec.tasks[] | select(.name | test("^wait-for-")) in operator-bundle-pipeline.yaml).
+  #
+  # 2. the overall timeouts for the respective image-building PipelineRuns.
+  #
+  # For example, in main-build.yaml the maximum timeout for building main images is set to 2h40m.
+  # Hence we set the tasks timeout here to 3h, because we need to give extra time for assembling
+  # the operator-bundle.
```

Comment:
In the same spirit, I suggest making this info more instructive.

```
  # When building the operator-bundle, we must resolve image digests for all other ACS product images in order to inject
  # these digests in the bundle's ClusterServiceVersion file. For this, the bundle pipeline waits for each image to be
  # built, i.e. its pipeline to complete.
  #
  # You must make sure:
  #
  # 1. For every `wait-for-image` task in `operator-bundle-pipeline.yaml` the `timeout` is equal to the timeout of the
  #    corresponding pipeline.
  #    E.g. if `main-build.yaml` has `spec.timeouts.pipeline` set to `2h40m`, make `wait-for-main-image`'s `timeout`
  #    (in `operator-bundle-pipeline.yaml`) also `2h40m`.
  # 2. The timeout for the operator-bundle build itself must be the maximum timeout of `wait-for-image` tasks plus
  #    30 minutes. Set this value in `spec.timeouts.tasks` (here in `operator-bundle-build.yaml`). Set
  #    `spec.timeouts.pipeline` to be `spec.timeouts.tasks` plus 10 minutes.
  #    E.g. if `wait-for-main-image` is the longest and has timeout `2h40m`, operator-bundle's `spec.timeouts.tasks`
  #    becomes `3h10m` and `spec.timeouts.pipeline` should be `3h20m`.
```

Reply:
commit 3dd479a21ee5d3a91301a132c7b356cc739d91f1